src/sql/workbench/contrib/executionPlan/browser/constants.ts

```diff
@@ -249,6 +249,14 @@ export let executionPlanNodeIconPaths =
 	unionAll: imageBasePath + 'union_all.png'
 };
 
+export const badgeIconPaths = {
+	warning: imageBasePath + 'overlay_warning.svg',
```

Comment:
Could probably just simplify this to `warning.svg` - or at least update to `badge_warning.svg`